### PR TITLE
102 not empty required props

### LINKED ISSUE #102: Wrong message when incorrectly specify the browser in the property
When I incorrectly specify name of browser in the properties file, I get " '' is not support yet". 
photo_2018-01-31_19-07-56

### PR DESCRIPTION
**Pull request by 0I0u3uk**
Fixed #102 

Watching after 112 MR, my other branch here

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 1]
```diff
@@ -808,6 +808,11 @@ public String getElementTitle(WebElement element) {
         for (Map.Entry<Field, String> entry : PageFactory.getPageRepository().get(this.getClass()).entrySet()) {
             try {
                 if (Core.getElementByField(this, entry.getKey()) == element) {
+                    if (entry.getKey().getAnnotation(ElementTitle.class) != null
```
Comment (PavelAlennikov) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: Should we extract entry.getKey().getAnnotation(ElementTitle.class) and entry.getKey().getAnnotation(ElementTitle.class).value() into local variables ?

File: src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java [hunk 2]
```diff
@@ -104,6 +104,10 @@ public static Actions getActions() {
      * @return the pagesPackage
      */
     public static String getPagesPackage() {
+        if("".equals(PAGES_PACKAGE)) {
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java]: use isEmpty() it always string

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java [hunk 3]
```diff
@@ -120,6 +120,10 @@ public static WebDriver getDriver() {
     }
 
     private static void createDriver() throws UnsupportedBrowserException, MalformedURLException {
+        if("".equals(WEBDRIVER_BROWSER_NAME)) {
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: use isEmpty() it always string

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java (hunk 1 above)
Comment (0I0u3uk) [reply src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: I don't think that it's good idea. Now it's look more elegantly IMHO

Comment (0I0u3uk) [reply src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: Fixed in https://github.com/sbtqa/page-factory/pull/112

**Comment by 0I0u3uk** [comment]
@clicman pls check PR

**Pull request merged**